### ISSUE TITLE
Cannot use import statement outside a module

### ISSUE THREAD

**Issue by code4you2021** [wxt-dev/wxt#132]
the error message:
```
import ReactDOM from "react-dom/client";
  ^^^^^^
SyntaxError: Cannot use import statement outside a module
```

content.tsx
```javascript 
import ReactDOM from "react-dom/client";
import Box from "../views/box";

export default defineContentScript({
  matches: ["<all_urls>"],
  runAt: "document_end",
  main() {
    const div = document.querySelector("#test-container");
    if (div) {
      ReactDOM.createRoot(div).render(<Box />);
    }
  },
});
```

Can you give a demo on how to mount react components in content.tsx?
I spent 2 hours and still can't get it to work, no problem with plasma!

Thanks.

**Comment by aklinker1** [comment]
This appears to be an issue with Jiti, which is used to load the entrypoint during the build to get the `matches` and `runAt` from the default export. See https://github.com/unjs/jiti/issues/138

I'll figure out a workaround for this.

**Comment by aklinker1** [comment]
Alright, I have a working solution, #134, but I need to get to work lol. I'll finish up the work sometime later today.

Once the fix is merged, you won't have to change anything, it will just work.

**Comment by aklinker1** [comment]
Released in [`v0.6.5`](https://github.com/wxt-dev/wxt/releases/tag/v0.6.5). Let me know if it doesn't work for you.

**Comment by code4you2021** [comment]
It's great that it's working properly.
thanks.
